.github/workflows/e2e/get_deployment_info/action.yaml

```diff
@@ -0,0 +1,43 @@
+name: Get Deployment Info
+description: Get deployment info from Astronomer API
+inputs:
+  deployment_id:
+    description: The deployment ID
+    required: true
+  organization_id:
+    description: The organization ID
+    required: true
+  astro_api_token:
```

Comment:
Are there any concerns about accidentally exposing tokens when it's an input?

Reply:
Github logging covers redacting secrets everywhere automatically, so we are safe unless Astro CLI exposes it

Reply:
Ah ok, how does it know this is a secret if it's just another input field?

Reply:
I think adding `::add-mask::VALUE` at the start of the workflow should mask the value from all the logs, but need to verify it. But answering your original question, it does it by secret's value I believe.